source/includes/_queries.md

```diff
@@ -40,13 +41,22 @@ Each query consists of following things:
     <li> **Query object** might be any of types, described below </li>
 </ul>
 
-**Signatures** contain one or many signatures (ed25519 pubkey + signature):
+**Signatures** contain one or many signatures (ed25519 pubkey + signature).
+
+## Validation
+
+The validation for all commands includes:
+
+ * timestamp — it should be in more than time of the peer minus 24 hours
```

Comment:
Maybe "shouldn't be older than 24 hours of the peer's time"?

Reply:
I would replace "shouldn't" to "should not", as documentation is quite formal, and "from the peer's time" is more grammatically correct.